qt/applications/workbench/workbench/app/mantidworkbench_launch_wrapper.py

```diff
@@ -0,0 +1,23 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantid workbench.
+import subprocess
+import sys
+
+from workbench.app.main import main
+
+
+def launch():
```

Comment:
This becomes much easier to add tests to if the `sys.argv` is an argument to the function
```suggestion
def launch(args):
```